.github/CODEOWNERS

```
@@ -0,0 +1,2 @@
+# Plugin Development
+/content/plugin/ @hashicorp/terraform-provider-devex
```

Team ID was updated this morning.

```suggestion
/content/plugin/ @hashicorp/terraform-devex
```